pkg/controller/elasticsearch/validation/validations.go

```diff
@@ -298,6 +298,26 @@ func validUpgradePath(current, proposed esv1.Elasticsearch) field.ErrorList {
 	return errs
 }
 
+func currentVersion(current esv1.Elasticsearch) (version.Version, *field.Error) {
+	// we do not have a version in the status let's use the version in the current spec instead which will not reflect
+	// actually running Pods but which is still better than no validation.
+	if current.Status.Version == "" {
+		currentVer, err := version.Parse(current.Spec.Version)
+		if err != nil {
+			// this should not happen, since this is the version from the spec copied to the status by the operator
```

Comment:
Is it not the version from the stored specification ? (which might be wrong if the validation webhook has been disabled at some point)

Reply:
I swapped the logic around and forgot to move the comment 😞